packages/angular/projects/ui-angular/src/lib/components/authenticator/components/form-field/form-field.component.ts

```diff
@@ -58,7 +58,7 @@ export class FormFieldComponent implements OnInit {
       this.authenticator.authState
     );
     const { validationError } = formContext;
-    return translate(validationError[this.name]);
+    return translate(validationError[this.name] as string);
```

Comment:
same here

<img width="758" alt="image" src="https://user-images.githubusercontent.com/11983489/157560078-a02defb1-d9e9-47b0-b1c4-731c9377b33f.png">

Reply:
Hmm. it was giving me an error in my VS Code, and it won't build.